wp-parsely.php

```diff
@@ -768,6 +768,28 @@ public function insert_parsely_page() {
 		return $parsely_page;
 	}
 
+	/**
+	 * Compare the post_status key against an allowed list (by default, only 'publish'ed content includes tracking data)
+	 *
+	 * @param  int|WP_Post $post  Which post object or id to check.
+	 * @return bool               Should the post status be tracked for the provided post's post_type. By default, only 'publish' is allowed.
+	 *
+	 * @see wp_parsely_track_status_slugs
+	 * @since 2.5.0
+	 */
+	public static function post_has_trackable_status( $post ) {
+		/**
```

Comment:
Consider adding a static variable in here, so that the filter only runs if the static variable isn't populated.

Reply:
Added a cache in 6a6349c.  Since this function accepts a post and can be re-run with another, I keyed the static cache on post_id.  Let me know what you think, @GaryJones

Reply:
@jblz Would it make more sense to have it keyed on the post_type instead?

Under what circumstances could a `foo` post type have different post statuses registered, for different instances/IDs of `foo`?

`register_post_status()` registered a status without assignment to any particular post type, but if https://core.trac.wordpress.org/ticket/12706 ever got completed, I think it would be associated with a post type, rather than individual instances of posts.